### PR TITLE
Refactor SE functionality after adding MetaData Channels scripting support

### PR DESCRIPTION
**Pull request by bogdanp-ni**
- [x] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/niveristand-instrument-addon-custom-device/blob/master/CONTRIBUTING.md).

### What does this Pull Request accomplish?

Refactor System Explorer by leveraging MetaData Channels scripting support.

### Why should this Pull Request be merged?

Reduces duplication, consolidating backend logic into scripting library.

### What testing has been done?

Local build and manual testing.

### CODE REVIEW

**Comment by bogdanp-ni** [comment]
Marked as draft, as it is dependent on another PR (https://github.com/ni/niveristand-instrument-addon-custom-device/pull/115).

**Comment by phutanu** [comment]
Add New MetaData Array.vi & Add New MetaData Channel.vi

- the "Assign New GUID to Refnum.vi" should be removed from the FP of both VIs as this node is already called inside the "Create MetaData Element.vi".

image
image

**Comment by phutanu** [comment]
I know we do not have yet a code guideline for how block diagrams should look like but I would like to see all the terminals viewed as icons for the below VI as this will improve greatly the readability.
image

**Comment by phutanu** [comment]
MetaData Channel.vi ->I do not want to be nitpicking and I also know you didn't touch this specific event cases, but during the refactoring and resizing the visual aspect of the code was affected. Could you please adjust this?

image
image

File: Source/Addon/Addon System Explorer/Instrument Addon System Explorer.lvlib [hunk 1]
```diff
@@ -110,4 +110,5 @@
 			<Item Name="Response Message Configuration.ctl" Type="VI" URL="../Types/Response Message Configuration.ctl"/>
 		</Item>
 	</Item>
+	<Item Name="Build Array Elements.vi" Type="VI" URL="../SubVIs/Build Array Elements.vi"/>
```
Comment (phutanu) [inline Source/Addon/Addon System Explorer/Instrument Addon System Explorer.lvlib:113]: This subVI should go under the "SubVIs" folder in the System Explorer.lvlib.

**Review by phutanu — CHANGES_REQUESTED** [review]
Please check my above comments.

**Comment by bogdanp-ni** [comment]
> MetaData Channel.vi -> I do not want to be nitpicking and I also know you didn't touch this specific event cases, but during the refactoring and resizing the visual aspect of the code was affected. Could you please adjust this?
> 
> image
> image

This makes a lot of sense :) I messed them up while rescaling the other event cases. Thanks!

**Review by phutanu — APPROVED** [review]

**Pull request merged**